### PR TITLE
fix benchmark_app python to support YES and NO values for -pin parameter

### PR DESCRIPTION
**Pull request by panhaiqi**
### Details:
 - *fix benchmark_app python to support YES and NO values for -pin parameter*

### Tickets:
 - *CVS-104390*

### CODE REVIEW

**Review by riverlijunjie — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
1. Do we need to the same change for C++ BA or this PR is about alignment?
2. Is it applicable for 2022.3?

**Review by nkogteva — APPROVED** [review]

**Comment by panhaiqi** [comment]
> 1. Do we need to the same change for C++ BA or this PR is about alignment?
> 2. Is it applicable for 2022.3?

C++ benchmark_app can support YES and NO values for -pin parameter now.

File: tools/benchmark_tool/openvino/tools/benchmark/main.py [hunk 1]
```diff
@@ -278,6 +278,10 @@ def set_throughput_streams():
             def set_nthreads_pin(property_name, property_value):
                 if property_name in supported_properties or device_name == AUTO_DEVICE_NAME:
                     # create nthreads/pin primary property for HW device or AUTO if -d is AUTO directly.
+                    if property_value == "YES":
```
Comment (peterchen-intel) [inline tools/benchmark_tool/openvino/tools/benchmark/main.py:281]: Please move this to the beginning of set_nthreads_pin and if property_name == 'AFFINITY':

**Comment by peterchen-intel** [comment]
> 1. Do we need to the same change for C++ BA or this PR is about alignment?
> 2. Is it applicable for 2022.3?

**Comment by peterchen-intel** [comment]
https://github.com/openvinotoolkit/openvino/pull/16042

**Pull request merged**